packages/flutter/lib/src/widgets/form.dart

```diff
@@ -349,6 +349,16 @@ class FormFieldState<T> extends State<FormField<T>> {
   /// True if this field has any validation errors.
   bool get hasError => _errorText != null;
 
+  /// True if the current value is valid.
+  ///
+  /// This will not set [errorText] or [hasError] and it will not update
+  /// error display.
+  ///
+  /// See also:
+  ///
+  ///  * [validate], which will update [errorText] or [hasError].
```

Comment:
Nit: Would this be more clear and correct? "which may update [errorText] and [hasError]."

Reply:
Done.